api/models/ssl.go

```diff
@@ -74,11 +74,14 @@ func CreateSSL(app, process string, port int, body, key string, chain string, se
 		return nil, fmt.Errorf("process does not expose port: %d", port)
 	}
 
-	arn, err := uploadCert(a, process, port, body, key, chain)
+	if arn == "" {
+		arn, err = uploadCert(a, process, port, body, key, chain)
 
-	if err != nil {
-		return nil, err
+		if err != nil {
+			return nil, err
+		}
 	}
+	// Check that the arn is valid?
```

Comment:
Could do some quick validation here that the string starts w/ `ARN::`, or just fall back on CloudFormation blowing up if you pass in an invalid ARN.